apps/web/app/(app)/gardens/[chain]/[garden]/pool/[poolId]/proposals/[proposalId]/page.tsx

```diff
@@ -215,71 +219,65 @@ export default async function Proposal({
-              {title}
-            </h3>
-          </div>
-          <div className="">
-            <p className="text-md text-justify">{description}</p>
-          </div>
+        <div className="flex w-full flex-col gap-8">
           <div>
-            {/* reqAmount - bene - creatBy */}
-            <div className="flex justify-between ">
-              {!isSignalingType && !!requestedAmount && (
-                <div className="flex flex-1 flex-col items-center space-y-4">
-                  <span className="text-md font-bold underline">
-                    Requested Amount
-                  </span>
-                  <span className="flex items-center gap-2 text-lg">
-                    {formatTokenAmount(requestedAmount, 18)}{" "}
-                    <span>{tokenSymbol}</span>
-                  </span>
-                </div>
-              )}
-              {!isSignalingType && beneficiary && (
-                <div className="flex flex-1 flex-col items-center space-y-4">
-                  <span className="text-md font-bold underline">
-                    Beneficiary
-                  </span>
-                  <EthAddress address={beneficiary} actions="copy" />
-                </div>
-              )}
-              {submitter && (
-                <div className="flex flex-1 flex-col items-center space-y-4">
-                  <span className="text-md font-bold underline">
-                    Created By
-                  </span>
-                  <EthAddress address={submitter} actions="copy" />
-                </div>
-              )}
+            <div className="mb-4 flex flex-col items-center justify-between gap-4 sm:mb-2 sm:flex-row sm:gap-2">
+              <h2>
+                Proposal #{proposalIdNumber} - {title}
+              </h2>
+              <Badge type={proposalType} />
+            </div>
+            <div className="flex items-center justify-center gap-4 sm:justify-start">
+              <Badge status={status} />
+              <p className="font-semibold">
+                {prettyTimestamp(proposalData?.createdAt || 0)}
+              </p>
             </div>
           </div>
+          <p>{description}</p>
+          <div className="flex flex-col gap-2">
+            {!isSignalingType && (
+              <>
+                <Statistic
+                  label={"requested amount"}
+                  icon={<InformationCircleIcon />}
+                >
+                  <DisplayNumber
+                    number={formatUnits(requestedAmount, 18)}
+                    tokenSymbol={tokenSymbol}
+                    compact={true}
+                    className="font-bold text-black"
+                  />
+                </Statistic>
+                <Statistic label={"beneficiary"} icon={<UserIcon />}>
+                  <EthAddress
+                    address={beneficiary}
+                    actions="copy"
+                    icon="identicon"
```

Comment:
the identicon looks awful :joy:

Reply:
I agree it not look  very nice, I can remove for now yeahp